project1/src/components/Comments.tsx

```diff
@@ -1,5 +1,12 @@
 import React from "react";
 import moment from "moment";
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
 
 interface commentItem {
```

Comment:
let's keep all the types in PascalCase
```suggestion
interface CommentItem {
```